### PR TITLE
Add effect on offer message highlight

### PR DESCRIPTION
**Pull request by axpoems**
Based on #2651

* Use a drop shadow instead of a solid border to highlight the offer message.
* Add an effect to remove the highlight after a few seconds.

### CODE REVIEW

File: apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/message_container/list/message_box/BubbleMessageBox.java [hunk 1]
```diff
@@ -360,4 +359,31 @@ protected static void onCopyMessage(ChatMessage chatMessage) {
...
+
+        showHighlightedPin = EasyBind.subscribe(item.getShowHighlighted(), showHighlighted -> {
+            if (showHighlighted) {
+                dropShadow.setRadius(15);
+                dropShadow.setSpread(0.3);
+                messageBgHBox.setEffect(dropShadow);
+                timeline.play();
+            } else {
+                timeline.stop();
+                messageBgHBox.setEffect(null);
```
Comment (HenrikJannsen) [inline apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/message_container/list/message_box/BubbleMessageBox.java:393]: Is that also applied at deactivate? If user switch screen while timeline is running and come back later we need to ensure there is no effect.

**Review by HenrikJannsen — APPROVED** [review]
utACK

File: apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/message_container/list/message_box/BubbleMessageBox.java (hunk 1 above)
Comment (HenrikJannsen) [reply apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/message_container/list/message_box/BubbleMessageBox.java:393]: Looked into src code more closely:
```
timeline.stop();
messageBgHBox.setEffect(null);
```
need to be added to the `dispose` method. timeline needs to be a fields and null checks are required in dispose....

Comment (axpoems) [reply apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/message_container/list/message_box/BubbleMessageBox.java:393]: Added the changes now.

**Review by HenrikJannsen — APPROVED** [review]
utACK

**Pull request merged**